llvm/test/Transforms/LoopVectorize/AArch64/armpl-calls.ll

```diff
@@ -1,21 +1,23 @@
-; RUN: opt -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize -S < %s | FileCheck %s --check-prefixes=CHECK,NEON
-; RUN: opt -mattr=+sve -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize -S < %s | FileCheck %s --check-prefixes=CHECK,SVE
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "call.*(cos|sin|tan|cbrt|erf|exp[^e]|gamma|log|sqrt|copysign|dim|min|mod|hypot|nextafter|pow|fma)" --version 2
```

Comment:
I think tests look OK, and the flags are consistent now:
- `-force-vector-interleave=1` is used in both NEON and SVE, and 
- `-prefer-predicate-over-epilogue=predicate-dont-vectorize` only for SVE.

The only thing I'm not sure is reverting back to the use of `exp[^e]` (for excluding calls like `llvm.experimental.*`

Reply:
Not sure what you mean, use of exp[^e] will exclude generating checks for things which start with "llvm.experimental.*", there is no use of such intrinsics in this file so there is no need to generate checks for it, to me all looks good, am I missing something?

Reply:
Oh right, so this expression is replaced in all 4 test files. In some of those it's needed for 'llvm.experimenta.*'.

Reply:
The [^e] regex is removed in the latest revision.